tests/SprykerEcoTest/Zed/PunchoutCatalogs/_support/PunchoutCatalogsBusinessTester.php

```diff
@@ -86,4 +101,82 @@ public function retrieveConnectionPasswordFromVault(int $idPunchoutCatalogConnec
                 $idPunchoutCatalogConnection
             );
     }
+
+    /**
+     * @param int $idCompanyBusinessUnit
+     *
+     * @return \Generated\Shared\Transfer\PunchoutCatalogConnectionSetupTransfer
+     */
+    public function createPunchoutCatalogsConnectionSetupTransfer(int $idCompanyBusinessUnit): PunchoutCatalogConnectionSetupTransfer
+    {
+        return (new PunchoutCatalogConnectionSetupTransfer())
+            ->setLoginMode(static::CONNECTION_SETUP_LOGIN_MODE)
+            ->setFkCompanyBusinessUnit($idCompanyBusinessUnit);
+    }
+
+    /**
+     * @return \Generated\Shared\Transfer\PunchoutCatalogConnectionCartTransfer
+     */
+    public function createPunchoutCatalogsConnectionCartTransfer(): PunchoutCatalogConnectionCartTransfer
+    {
+        return (new PunchoutCatalogConnectionCartTransfer())
+            ->setDefaultSupplierId(static::CONNECTION_CART_SUPPLIED_ID)
+            ->setMaxDescriptionLength(static::CONNECTION_CART_MAX_DESCRIPTION_LENGTH)
+            ->setMapping(static::CONNECTION_CART_MAPPING)
+            ->setEncoding(static::CONNECTION_CART_ENCODING);
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\CompanyBusinessUnitTransfer $companyBusinessUnitTransfer
+     *
+     * @return \Generated\Shared\Transfer\PunchoutCatalogConnectionTransfer
+     */
+    public function createPunchoutCatalogConnection(CompanyBusinessUnitTransfer $companyBusinessUnitTransfer): PunchoutCatalogConnectionTransfer
+    {
+        $punchoutCatalogConnectionTransfer = $this->havePunchoutCatalogConnection([
+            PunchoutCatalogConnectionTransfer::NAME => static::CONNECTION_NAME,
+            PunchoutCatalogConnectionTransfer::USERNAME => static::CONNECTION_USERNAME,
+            PunchoutCatalogConnectionTransfer::PASSWORD => static::CONNECTION_PASSWORD,
+            PunchoutCatalogConnectionTransfer::TYPE => static::CONNECTION_TYPE,
+            PunchoutCatalogConnectionTransfer::FORMAT => static::CONNECTION_FORMAT,
+            PunchoutCatalogConnectionTransfer::FK_COMPANY_BUSINESS_UNIT => $companyBusinessUnitTransfer->getIdCompanyBusinessUnit(),
+            PunchoutCatalogConnectionTransfer::SETUP => $this->createPunchoutCatalogsConnectionSetupTransfer($companyBusinessUnitTransfer->getIdCompanyBusinessUnit()),
+            PunchoutCatalogConnectionTransfer::CART => $this->createPunchoutCatalogsConnectionCartTransfer(),
+        ]);
+
+        return $punchoutCatalogConnectionTransfer;
+    }
+
+    /**
+     * @return \Generated\Shared\Transfer\PunchoutCatalogTransactionTransfer
+     */
+    public function createPunchoutCatalogTransaction(): PunchoutCatalogTransactionTransfer
```

Comment:
`createPunchoutCatalogTransactionTransfer`?

Reply:
I do not think it's the good name for it because it creates `PunchoutCatalogTransaction` in persistence, but not a just transfer.